extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -84,6 +84,7 @@ export enum FileStorageType {
 }
 
 export enum Page {
+	ImportAssessment,
```

Comment:
after importing assessment, 'save and close' still work as expected?

Reply:
will fix this as part of bug: [Task 2812363](https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2812363): [ADS] 'save and close' bug

Reply:
Discussed this with Abhishek. Save and Close will work as expected after import. Session will resume from where user saved before starting import.